pkg/render/intrusion_detection.go

```diff
@@ -701,10 +707,15 @@ func (c *intrusionDetectionComponent) deploymentPodTemplate() *corev1.PodTemplat
 			})
 	}
 
-	container := relasticsearch.ContainerDecorateIndexCreator(
-		relasticsearch.ContainerDecorate(c.intrusionDetectionControllerContainer(), c.cfg.ESClusterConfig.ClusterName(),
-			ElasticsearchIntrusionDetectionUserSecret, c.cfg.ClusterDomain, rmeta.OSTypeLinux),
-		c.cfg.ESClusterConfig.Replicas(), c.cfg.ESClusterConfig.Shards())
+	container := c.intrusionDetectionControllerContainer()
+
+	envVars := []corev1.EnvVar{
```

Comment:
I don't think there's any need to keep these appended separately from the other env vars defined in `intrustionDetectionControllerContainer()`

Reply:
Agreed and this was a change I already had in my next set of changes to be pushed while I was fixing some issues I found once I loaded my images into a cluster.